ui/angular/src/app/measure/create-measure/raw/raw.component.html

```diff
@@ -0,0 +1,100 @@
+                1
+              </div>
+              <span class="stepDesc text-small"> Configuration </span>
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div id="step-1" class="formStep">
+        <label class="stepDesc">Please setup the measure required information</label>
+        <div class="container-fluid">
+          <div class="col-md-12 col-lg-12 col-sm-12">
+            <fieldset>
+              <legend>
+                Required Information
+              </legend>
+              <div class="col-md-12 col-lg-12 col-sm-12" style="margin-top:30px;">
+                <div class="form-group"
+                     [ngClass]="{'has-error': !valid, 'has-success': valid}">
+                  <div class="row">
+                    <label class="col-md-2 col-lg-2 col-sm-2 control-label">
+                      Raw measure<span class="symbol required"></span>:
+                    </label>
+                    <div class="col-md-10 col-lg-10 col-sm-10">
+                      <div class="btn-group btn-group-sm navbar-right" role="group" aria-label="Format">
+                        <button type="button" class="btn btn-primary" [ngClass]="{'active': format == Format.json}" (click)="changeFormat(Format.json)">JSON</button>
+                        <button type="button" class="btn btn-primary" [ngClass]="{'active': format == Format.yaml}" (click)="changeFormat(Format.yaml)">YAML</button>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="row">
+                    <textarea class="code-viewport" rows="20" required [(ngModel)]="data" #content="ngModel" name="content" (ngModelChange)="onInputChange()"></textarea>
+                    <span class="error text-small block " *ngIf="!valid">Measure is not valid</span>
+                  </div>
+                </div>
+              </div>
+              <div style="color:#b2c831">
+                <p>
+                  <i class="fa fa-info-circle"></i> After submitted, please go to "<a class="bark-link"
+                                                                                      routerLink="/measures">Measures</a>"
+                  to check the measure status
+                </p>
+              </div>
+            </fieldset>
+          </div>
+          <div class="form-group btn-container">
+            <toaster-container></toaster-container>
+            <button type="submit" (click)="submit(prForm)" class="btn btn-primary btn-o next-step btn-wide pull-right">
+              Submit
+            </button>
+          </div>
+        </div>
+      </div>
+      <div class="modal fade" id="confirm" role="dialog" #modal tabindex="-1" [ngClass]="{'in': visibleAnimate}"
+           [ngStyle]="{'display': visible ? 'block' : 'none', 'opacity': visibleAnimate ? 1 : 0}"
+           (click)="onContainerClicked($event)">
+        <div class="modal-dialog modal-xg modal-lg">
```

Comment:
this dialog looked not centered on screen record -- is that true or it's just me?
can you make it appear on center of the screen?

Reply:
I just didn't captured whole window, I've just checked and it's centered.

Reply:
Oh ok. Thanks for checking!